docs/source/examples.rst

```diff
@@ -38,6 +38,7 @@ Computed Tomography
    examples/ct_astra_odp_train_foam2
    examples/ct_astra_unet_train_foam2
    examples/ct_projector_comparison
```

Comment:
We probably have other examples of this elsewhere, but it would look a little better to have `_2d` and `_3d` postfixes rather than nothing and `_3d`.

Reply:
Done.